docs/rackhd/configuration.rst

```diff
@@ -170,6 +170,268 @@ configuration. Custom certificates can also be used with some configuration.
+                'set-bmc-credentials-graph': 'succeeded'
+            },
+            taskName: 'Task.Catalog.GenerateSku'
+        },
+    
+  
+Next, edit **Discovery workflow graph** located at ``on-taskgraph/lib/graphs/discovery-graph.js``
+to remove the reboot task. The reboot task is already included in the **set-bmc-credentials-graph** 
+that was added to the **Sku Discovery graph** in the first step.
+Below is a snippet of the Discovery graph without the reboot task (the reboot task was originally located
+after the task 'catalog-lldp')
+
+.. code-block:: javascript
+
+   module.exports = {
+    friendlyName: 'Discovery',
+    injectableName: 'Graph.Discovery',
+    options: {
+        'bootstrap-ubuntu': {
+            'triggerGroup': 'bootstrap'
+        },
+        'finish-bootstrap-trigger': {
+            'triggerGroup': 'bootstrap'
+        }
+    },
+    tasks: [
+        {
+            label: 'bootstrap-ubuntu',
+            taskName: 'Task.Linux.Bootstrap.Ubuntu'
+        },
+        {
+            label: 'catalog-dmi',
+            taskName: 'Task.Catalog.dmi'
+        },
+        {
+            label: 'catalog-ohai',
+            taskName: 'Task.Catalog.ohai',
+            waitOn: {
+                'catalog-dmi': 'finished'
+            }
+        },
+        {
+            label: 'catalog-bmc',
+            taskName: 'Task.Catalog.bmc',
+            waitOn: {
+                'catalog-ohai': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-lsall',
+            taskName: 'Task.Catalog.lsall',
+            waitOn: {
+                'catalog-bmc': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-megaraid',
+            taskName: 'Task.Catalog.megaraid',
+            waitOn: {
+                'catalog-lsall': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-smart',
+            taskName: 'Task.Catalog.smart',
+            waitOn: {
+                'catalog-megaraid': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-driveid',
+            taskName: 'Task.Catalog.Drive.Id',
+            waitOn: {
+                'catalog-smart': 'finished'
+            },
+            ignoreFailure: true
+        },
+        {
+            label: 'catalog-lldp',
+            taskName: 'Task.Catalog.LLDP',
+            waitOn: {
+                'catalog-driveid': 'finished'
+            },
+            ignoreFailure: true
+        },
+       {
+            label: 'finish-bootstrap-trigger',
+            taskName: 'Task.Trigger.Send.Finish',
+            waitOn: {
+                'catalog-lldp': 'finished'
+            }
+        }
+    ]
+   };
+
+
+Once the above steps are completed (edited and saved) the services need to be restarted:
+
+.. code-block:: shell
+
+    sudo service on-http start
```

Comment:
Why do all of the services require a restart?  Should only need to restart on-taskgraph right?